etherparse/src/net/ipv4_ecn.rs

```diff
@@ -16,7 +16,7 @@ impl Ipv4Ecn {
     pub const TWO: Ipv4Ecn = Ipv4Ecn(2);
 
     /// Ipv4Ecn with value 0.
-    pub const TRHEE: Ipv4Ecn = Ipv4Ecn(3);
+    pub const THREE: Ipv4Ecn = Ipv4Ecn(3);
```

Comment:
```suggestion
    pub const THREE: Ipv4Ecn = Ipv4Ecn(3);

    #[deprecated(
        since = "0.18.0",
        note = "Please use the type Ipv4Ecn::THREE instead."
    )]
    /// Deprecated, use [`crate::Ipv4Ecn::THREE`] instead.
    pub const TRHEE: Ipv4Ecn = Ipv4Ecn::THREE;
```